modmesh/app/euler1d.py

```diff
@@ -839,3 +839,6 @@ def load_app():
     config_window.redraw()
     plotting_area.redraw()
     _subwin.show()
+
+# vim: set ff=unix fenc=utf8 et sw=4 ts=4 sts=4: 
+
```

Comment:
<img width="553" alt="image" src="https://github.com/user-attachments/assets/8745b29b-669c-4f68-b89e-a666907f5849">

In the editor, it is clear that there are two newlines after the vim modeline, and the github action should fail under this circumstance.
Thus, you may modify this part according to the following rules:
1. There should be only a newline after the vim modeline.
2. The newline should not contain any character, such as the tab `'\t'` and the space ` `.